templates/generic-apply-rule-template.conf.j2

```diff
@@ -0,0 +1,30 @@
+// {{ ansible_managed }}
+{% import 'object_attributes.j2' as attributes %}
+
+{% set target = item.value.pop('target', None) %}
+{% set iterator = item.value.pop('iterator', None) %}
+{% set operators = item.value.pop('var_operators', None) %}
+
+{% for rule_name, params in item.value.items() %}
+apply {{ item.key }} "{{ rule_name }}" {% if iterator is defined and iterator is not none -%}
+for ({{ iterator }})
+{%- endif -%}{% if target in ['Host', 'Service'] %} to {{ target }} {% endif -%}
```

Comment:
As I mentioned, I'd really appreciate some help/guidelines with vertical and horizontal spacing in jinja.